app/src/main/java/com/example/androidthings/imageclassifier/ImageClassifierActivity.java

```diff
@@ -72,19 +75,44 @@ protected void onCreate(final Bundle savedInstanceState) {
         mResultViews[0] = (TextView) findViewById(R.id.result1);
         mResultViews[1] = (TextView) findViewById(R.id.result2);
         mResultViews[2] = (TextView) findViewById(R.id.result3);
+        View takePhotoButton = findViewById(R.id.takePhoto);
+
+        if (!isAndroidThingsDevice(this)) {
+            // Android Things devices have an LED and a hardware button.  On other devices (like
```

Comment:
Android Things devices don't necessarily have an LED and a hardware button, and this onscreen button can be useful if a user is running the app on an AT board without special buttons/LEDs. I would always show it, even if it's AT.

Reply:
My display isn't a touchscreen, so I found it confusing to show a button that you can't touch to press.  Are you sure you want it displayed even when you can't press it?  Is there a better test I can do here?

Reply:
You can still connect a pointing device, like a USB mouse, but I see your point. Perhaps you can remove the button and take a photo when the user taps on the photo area, so that the UI is not confusing to users without touching screens or pointing devices, but still work well for users with a touchscreen.